options.go

```diff
@@ -0,0 +1,24 @@
+package goshopify
+
+import "fmt"
+
+// Option is used to configure client with options
+type Option func(c *Client)
```

Comment:
Thank you for moving this to the separate file.

Reply:
my pleasure